src/js_native_api.h

```diff
@@ -289,7 +289,7 @@ NAPI_EXTERN napi_status NAPI_CDECL napi_instanceof(napi_env env,
 
 // Gets all callback info in a single call. (Ugly, but faster.)
 NAPI_EXTERN napi_status NAPI_CDECL napi_get_cb_info(
-    napi_env env,               // [in] NAPI environment handle
```

Comment:
Is `napi_get_cb_info` needed in no-gc callbacks? `napi_callback_info` should only be used in native callback of JS functions, which implies that GC can interleave in the call.

Reply:
I moved this one back to `napi_env`.